src/sth_app.js

```diff
@@ -39,22 +41,37 @@
       }
 
       // Connection to the MongoDB database successfully established
-      sthLogger.info('Connection to MongoDB %s successfully established',
-        sthDatabase.connectionURL);
+      sthLogger.info(
+        'Connection to MongoDB %s successfully established',
+        sthDatabase.connectionURL,
+        {
+          operationType: sthConfig.OPERATION_TYPE.DB_CONN_OPEN
+        }
+      );
 
       // Start the hapi server
       sthServer.startServer(
         sthConfig.HOST, sthConfig.PORT, sthDatabase, function (err) {
           if (err) {
-            sthLogger.error(err);
+            sthLogger.fatal(err.toString(), {
+              operationType: sthConfig.OPERATION_TYPE.SERVER_START
+            });
           } else {
-            sthLogger.info('Server started at', sthServer.server.info.uri);
+            sthLogger.info('Server started at', sthServer.server.info.uri, {
+              operationType: sthConfig.OPERATION_TYPE.SERVER_START
+            });
           }
         });
     }
   );
 
+  // In case Control+C is clicked, exit gracefully
   process.on('SIGINT', function () {
     return exitGracefully(null);
   });
+
+  // In case of an uncaught exception exis gracefully
```

Comment:
Fixed in https://github.com/telefonicaid/IoT-STH/commit/02fb9825d614d8937e22617710b8f94c2fef13dd